### PR TITLE
The Great Refactoring (+ doc updates)

### PR DESCRIPTION
**Pull request by Ferdinand737**
# Description

- Removed all traces of cypress
- Removed unused files

# Questions
- Can we remove contents of [/.github/scripts](https://github.com/ubco-db/office-hours/tree/master/.github/scripts)?

Feel free to commit any other removals to this PR

### CODE REVIEW

File: packages/server/.env.development [hunk 1]
```diff
@@ -4,9 +4,6 @@ PUBLICKEY=BICM9ljt6UzPC2HsKwlGotdrOSjva59TcwEnolgjf67P8BLN-UStYY2kvTyKDT5DpqJX0s
 PRIVATEKEY=3134tddi2nyoxnamvbImuban4fXXgiiXia4t3SGVOBA
 SESSION_SECRET=chunguschunguschunguschunguschunguschunguschunguschungus
 EMAIL=mailto:zuck@fb.com
-TWILIOACCOUNTSID=ACREMEMBERTHEALAMO
-TWILIOAUTHTOKEN=FLYMETOTHEMOONANDLETMESINGFOREVERMORE
-TWILIOPHONENUMBER=+14204206969
 JWT_SECRET=chunguschunguschunguschunguschunguschunguschunguschunguschungus
```
Comment (Ferdinand737) [inline packages/server/.env.development:7]: Should we change this to something less silly?

File: packages/server/src/mail/mail.module.ts [hunk 2]
```diff
@@ -34,6 +35,7 @@ export class MailModule {}
       // Use an empty class for a mock implementation
       useValue: {
         sendUserVerificationCode: () => 'fake code',
+        sendEmail: (_emailPost: sendEmailAsync) => 'fake email',
```
Comment (AdamFipke) [inline packages/server/src/mail/mail.module.ts:38]: what does this do?

Comment (wskksw) [reply packages/server/src/mail/mail.module.ts:38]: I don't think I did that. But it seems to serve a purpose from the comments rn. I think we can leave it in?

**Review by AdamFipke — APPROVED** [review]
reviewed it. Looks good. I didn't test if the notification changes broke anything but I'm gonna hope that it didn't

**Pull request merged**